sts3x/config.inc

```diff
@@ -0,0 +1,25 @@
+-include build_config.inc
+sensirion_common_dir ?= ../embedded-common
+sts_common_dir ?= ../sts-common
+sts3x_dir ?= ./
+CONFIG_I2C_TYPE ?= hw_i2c
+
+sw_i2c_impl_src ?= ${sensirion_common_dir}/sw_i2c/sensirion_sw_i2c_implementation.c
+hw_i2c_impl_src ?= ${sensirion_common_dir}/hw_i2c/sensirion_hw_i2c_implementation.c
+
+CFLAGS ?= -Os -Wall -fstrict-aliasing -Wstrict-aliasing=1 -Wsign-conversion -fPIC
+CFLAGS += -I. -I${sensirion_common_dir} -I${sts_common_dir} -I${sts3x_dir} \
+          -I${sensirion_common_dir}/${CONFIG_I2C_TYPE}
+
+sensirion_common_sources = ${sensirion_common_dir}/sensirion_arch_config.h \
+                           ${sensirion_common_dir}/sensirion_i2c.h \
+                           ${sensirion_common_dir}/sensirion_common.h \
+                           ${sensirion_common_dir}/sensirion_common.c
+sts_common_sources = ${sts_common_dir}/sts_git_version.h \
+                     ${sts_common_dir}/sts_git_version.c
```

Comment:
This Makefile isn't standalone anymore since it doesn't know how to build the `sts_git_version.c`:
```
% cd sts3x && make
make: *** No rule to make target '../sts-common/sts_git_version.c', needed by 'sts3x_example_usage_hw_i2c'.  Stop.
```

Reply:
But I guess this was already an issue before.

Reply:
yes, nothing changed there. I though about adding

```
prepare:
    cd .. && $(MAKE) prepare
```

but it adds more noise and is not reliable, e.g. on the release build it'd need to be removed manually which is even worse.

Reply:
This is an independent issue since it was present before.